src/Data/GeneratorMain.purs

```diff
@@ -62,9 +62,8 @@ generatorMain :: Effect Unit
 generatorMain = launchAff_ do
   (Args args) <- liftEffect $ execParser opts
   errs <- generatePS args
-  when (A.null errs)
-    $ liftEffect
-    $ exit 1
+  liftEffect <<< exit $
+    if (A.null errs) then 0 else 1
```

Comment:
actually this just reminds me, we should really be logging these errors:

https://github.com/f-o-a-m/purescript-web3-generator/issues

Reply:
We actually already log them, in `generatePS`: https://github.com/f-o-a-m/purescript-web3-generator/blob/purs-0.15/src/Data/CodeGen.purs#L68-L72